### DISCUSSION TITLE
Thinking Cap #2

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/211A-ASIAAM-191A#7] [Assignments]
#### Note: responses submitted after Tuesday 1pm (7/6) will not be eligible for awards.

> ## How can participatory mapping and critical cartography be improved?

### Submission
Post your [thinking cap #2](https://github.com/albertkun/211A-ASIAAM-191A/blob/master/Week_02/Materials/thinking-cap.md) response here.

**Comment by cynthinee** [comment]
I appreciate the participation ladder with roles and responsibilities (p. 9) in the Chambers article. The extractive model is really prevalent in academic, in which the researcher extracts knowledge from the community they study and then get all the credit for the knowledge produced. I think about contesting authorship a lot in scholarly publications. You don't see community members as authors nearly enough in my opinion. With that said, as a PhD student, one thing I really don't like about the dissertation is that ultimately I'm a single author of the dissertation and my participants at most can be a collaborator. I talk with a lot of other graduate students about our post-UCLA plans to continue our research in more meaningful ways to communities. 

I think something I would like to learn about in participatory mapping is how to incorporate offline technologies to co-produce maps. I did an exercise in a spatial ethnography workshop in winter where we walked around our neighborhoods and just kept notes and mapped things through careful looking, which I really dug. 

Here's my final spatial ethnography map (sorry for the bad art):

![IMG_2953](https://user-images.githubusercontent.com/31974707/124519652-caaf7600-dd9e-11eb-9acc-58b29b306682.jpg)

**Reply by abi-200** [reply]
The idea you bring up of claiming false ownership is very eye-opening to me - in my time in academia thus far, I have read many research papers with intricate experiments and data collection, and yet the paper is credited to one or a few authors. This is something that I have not personally thought about before now, but I can now think more critically about how information transfer itself needs to be carefully considered/monitored to make sure that it is following fair use/credit (in other words, how every step of data collection should be held to this standard). I also enjoyed seeing your offline map creation for your spatial ethnography workshop - I think this is a creative way to gain experience making maps with more accessible tools/information than is available to the average person, and the process of creating a map in this way can allow one to pay more attention to their surroundings/community than they may otherwise be inclined to do over a screen program (where much information can be found without needing to explore the community at hand).

**Reply by luigimagdamit** [reply]
I think your point about authorship in research is something that is new to me. I think having one person claim ownership over research that is gathered through a community goes against the ethos of participatory mapping, and I think its really admirable that you are trying to find ways to overcome it. The map is also well drawn!

**Reply by ronnie-blanche** [reply]
Hey, Cynthia! I found it really thoughtful how you pointed out that community members are often overlooked and forgotten when it comes to recognizing their efforts in academia. As we addressed privilege in last week's lecture, education and academia are both forms of power and advantages. I never knew that these sort of issues existed within research and authorship in the academic field, but it is still a large issue as community members deserve recognition of their contributions otherwise this research to help liberate them continues to oppress them.

**Comment by abi-200** [comment]
Although this viewpoint might be somewhat idealist, I believe that one major way participatory mapping can be improved is by dismantling academic elitism when it comes to the practice of cartography. As the Chambers article points out, up until the 1980s/1990s, many academics held the notion that indigenous and local people not part of their trained circle could not create useful, efficient maps as well as they themselves could. Locals were treated more as tools for themselves to create maps, rather than as creators themselves. Later on, it was realized that local people were able to quickly (within a few hours) create maps showcasing their own specific communities (ex. Well locations, farming purposes) with high quality, and the article also later shows that this is made possible when participants are made to feel confident in their own abilities by a facilitator. While this article was written in 2005, and since then more community mapping applications and courses have become more available, the problem itself still seems prevalent such that these opportunities are still largely limited to people with the privilege of access to academia (ex. University students taking courses, people with access to GIS equipment and networks that let them know about mapping technologies, etc). In this same regard, improvement of participatory mapping and critical cartography can also be made by giving value to maps that do not strictly adhere to the western viewpoints of what a map is. For example, in the article I learned that ground maps are still prevalent in some areas of the world, and had we not recognized these maps as legitimate, we would have lost out on valuable map information given by a group that we should hope to empower, thereby keeping the existing power structure (facilitator over participants) in place.
Another way participatory mapping and critical cartography can be improved is by giving more care to the well-being of the participants; if participatory mapping is to be used to raise the voices of a community with their own words and views, the process of mapping itself should not bring harm to participants as much as can be controlled. Just as we have provisions in place to protect research participants in academic settings (ex. IRB guidelines), a similarly strict set of universal guidelines should be set in place to protect participants (particularly those in protected classes or vulnerable positions) - for example, participants should be given clear expectations with how their participation will help, their participation should benefit them to some degree, they should be the ones in control, etc.

**Reply by luigimagdamit** [reply]
I think your point about indigenous and non Western methods of mapping is really interesting. It makes me wonder how much knowledge has been either lost or overwritten in the past due to the removal of many cultural knowledge systems.  It reminds me of the different ways mathematics is represented across different cultures, and how many conventions can differ across these cultural representations of math, which is usually seen as something very objective.

**Reply by cynthinee** [reply]
In my qualitative methods course, my professor constantly repeated, "The IRB is just the **minimum** guidelines for ethical research" and we were really challenged to think about what ethics and care look like in our research projects. I don't know if other folks in this class have been a study participant before, but the last time I was a participant for someone's PhD research it was kind of the worst. They were really nice to me leading up to the interview, but as soon as the interview was over they just left and I never heard from them again. Like, not even a thank you email. 

I really like that you use the word 'care' in thinking about participatory mapping and critical cartography. I think that goes beyond academic ethics and really center communities and what they want and need.

**Reply by hrfrahn** [reply]
You made a great point about how mapping skills and GIS training opportunities, although becoming more widespread, are still mainly limited to those in academia (like us in this class!). Participatory mapping would be much improved if these skills were more common; this would allow anyone to produce the maps they need without relying on outside "professional" help.

**Comment by luigimagdamit** [comment]
I think that participatory mapping can be accompanied by the development of more tools that give communities a medium in which they can share their collective spatial knowledge. I believe that one example of this is the navigation app “Waze” which allows users to put markers down for events such as car crashes or police on the road. This allows people to give others in their community knowledge about the surrounding roads, and enables me to break the speed limit in a suburban area without a second thought. 

I believe that critical cartography can be addressed by spreading more awareness of other methods of spatial technology. This way, we can explore other methods of visualizing our surrounding space and develop a more collective and democratized sense of this space rather than depending on maps that are designed by a select few.

**Reply by abi-200** [reply]
I agree with your point about communities being given tools that they are able to use in order to empower themselves by sharing knowledge - I actually had not heard of the Waze app, but I searched it on the App Store after and I think it's really important that such a tool exists! It does bring up the question of how to get more people to know about/use the app, especially for people without access to smartphones or easy computer access to reach it, but overall I think it's cool that such a user-based map app is so popular.

**Reply by edmondywen** [reply]
I really like the example of Waze. It helps to contextualize the practice of community sourced information in an everyday use case that many people are aware of. I also appreciate the idea of spreading knowledge in the second paragraph so that spatial technology is more accessible to everyone.

**Reply by perezsara59** [reply]
I really like how you mentioned the idea of a medium to share collective knowledge because who knows a community better than the people that live there! The medium could be used to share restaurants and help small businesses but also you could see communities through the collective lens of its residents. Everyone has a different experience and different spots that they would consider important to their community so seeing those spots that other mapping mediums may miss would be really cool.

**Comment by eandrusenki** [comment]
Some ways that participatory mapping and critical cartography can be improved is by shifting power and control away from those who are participating AND have a sense of academic or intellectual superiority, and in the article is described somewhat as “tyranny” of participation. I think there needs to be some sort of bias training, if you could call it that where we can remove ignorance from the eyes of facilitators and teach them what spatial tools and technology means all over the world and see how other people navigate space, and if they choose to navigate space differently, that doesn't mean their spatial tools don't constitute a map. As mentioned in the article and by our guest speaker Mariah, there needs to mainly be a shift away from maps appealing to the western view. 
I think that participatory mapping should not be "ruled" by those with those most power and the most (western) skills, but rather it changes from community to community depending on who is being helped, that way voices who need help and need to get a message out are being amplified instead of muffled by those in charge.

**Reply by abi-200** [reply]
Bias training sounds like it could be a great idea if we can educate the next generation(s) of facilitators to not depend on western viewpoints. In confining mapping and cartography to this view, we lose out on a lot of valuable tools and information about the communities we hope to help, and this opens up the potential for the facilitator to fall into the problematic savior trope. You also bring up an interesting point about who is in power changing from community to community, and that there is no singular answer - I wonder if since our goal is to empower community members to be active participants and owners of their own information, if it is necessary to have one "ruler" in charge at all.

**Reply by edmondywen** [reply]
I really like the emphasis on shifting power and control away from established institutions. While challenging status quo systems is a monumentally difficult task, focus on improving the everyday conditions for everyday people is certainly the way to go. For example, I agree that it is necessary for us to uplift the voices of people who need help and to focus on the specific needs of various communities so that everyone gets a seat at the table.

**Reply by ramirezirvin** [reply]
I totally agree with the shift in power from the academics and scholars. While they do try their best to improve it for their benefit their improvements do tend to disadvantage the original creators of the maps themselves because of their lack of resources. And the work done should not be claimed by those who are in most power but help the original creators themselves be the one's in power and decide upon the new changes to be made.

**Comment by edmondywen** [comment]
One interesting point in the Chamber's paper that stuck out to me is the note on how mapping projects are heavily dependent on everyday people and yet oftentimes fail to respect their time and effort. A reason given is that “common professional belief” assumes that poor people’s time is unimportant when it is actually vitally important to their livelihoods especially during “difficult times of year” (Chambers 2017).  This note highlights the negative impact of power imbalances inherent to research because when researchers and participants have different goals and needs, those of the researchers, not community members, are prioritized. This furthers the extractive nature of GIS research and academia, damaging relationships for generations to come. To address this issue, it is vital that we understand and acknowledge how power imbalances can harm the project and most importantly, the people. Moreover, the research should be built to prioritize the people’s needs to begin with so that these issues can be confronted from the foundation of the project itself.

**Reply by perezsara59** [reply]
This is definitely true and something I have seen happen in my community. I have increasingly seen my neighbors become less well-receiving of people who come and ask questions because they feel like they are just taking their information. Additionally, when they never see the outcome or product of what their opinion was used for it can add to that sense that perhaps they were ignored or their answers weren't as valuable so they went unused.

**Reply by emilysuh** [reply]
I agree with everything you say here and really like the example you brought up of researchers not respecting their research participants' time. Time is a necessary component of developing trust and relationship between the researcher and the community members, but it is highly undervalued in research institutions because of their desire for quick deliverables. Figuring out how to share power and allow for self-determination of participants in the specific contexts they are working from takes much time and effort.

**Reply by hrfrahn** [reply]
I think your point about the power imbalances between researchers and participants is really important. There are often already inherent power imbalances when a researcher comes into any community and expects cooperation, but asking participants to spend hours making maps for little/no reward is unrealistic and honestly insulting. Researchers seeking community-sourced maps should provide some form of compensation or at least ensure that the map being produced is clearly and immediately beneficial to the community it's sourced from.

**Comment by perezsara59** [comment]
Participatory mapping needs community input. Language barriers often prevent or discourage people from participating, but even when translators are available some people can be weary of providing strangers with such information/opinion. I think its important for the critical cartographer to build trust with the community they're working with and possibly even find a common ground or something they can relate to with the community. This way it doesn't feel like some stranger taking information from you to use it for something you don't quite understand but rather, they know their role and their importance in this project and they know their voice/opinion is valuable, also informing them and involving them in the process as much as you can explaining what it is you're doing so they know why you need this data from them. Participatory mapping and critical cartography can also be improved by capturing all different kinds of experiences and stories, being more inclusive. However in order to do this I think the resources to map need to be shared more with others. It can be intimidating to learn something new but that becomes even more difficult when you don't have access to the resources that grant you these opportunities. 

Personally in my experience, I was introduced to GIS and map making in college. Learning these skills and thinking about the different ways I could learn more about my community and translate it into something anyone can understand visually regardless of a language barrier was really encouraging and pushed me to continue to pursue GIS and learn more. That being said, there are definitely times when I need to reflect on what I am doing and why I am portraying things a certain way because I don't want to contribute to something that could potentially be harmful. 

The Chambers article touches on which maps were considered useful which is also something Mariah mentioned in presentation when she asked us what we considered to be a map. I think this is something very important to keep in mind to broaden not only my thinking but others as well towards what they consider to be a map.

**Reply by eandrusenki** [reply]
I think that's a really good point you bring up that it's important for critical mappers to build a community with or become part of the community in which they are doing work with. I think it would be a really good way to flush out any preconceived ideas or biases that the critical mapper originally has. I also am glad you brought up and pointed out language barriers as being an issue because most of us have focused on where control is but no one has really mentioned ways in which people can automatically be excluded from participating based on accommodations not being properly made.

**Reply by emilysuh** [reply]
I agree with everything you've said here! I found your point about language accessibility really interesting as a bilingual who is interested in organizing with non-English language communities. I think it would be really cool if a lot of maps had translations based on the languages spoken by the community and devote some of the project's resources toward language accessibility.

**Comment by emilysuh** [comment]
Participatory mapping and critical cartography can be improved by emphasizing relationship building as one of its core goals. Does the project make room and structures for participants to connect with each other and strengthen relationships sustainably over the long-term, and what kind of strategic alliances can be formed across difference?  How does the project make room for people to co-learn together and encourage everyone to take collective ownership for their future? Additionally, capacity building among community members would help democratize participatory action research -- that is, teaching more people how to design and conduct participatory action research and mapping would mean more people being empowered to craft their own projects rather than being a passive participant in others' research. Skills building in this way would be a sincere attempt at sharing power among community members and researchers with more formalized education or institutional affiliations.

**Reply by eandrusenki** [reply]
I am really glad that you and Sara mentioned this idea of being "a part of" the community and relationship building needing to be a goal for critical cartography and participatory mapmaking. And that a main priority be making room for members of the community to be able to participate and making sure that voices will be amplified, everyone gains something, the relationship is equal (someone is not giving more, someone else is not losing more) and I also agree that we should not allow people to be passive participants and giving them the skills to have power in their own projects.

**Reply by ronnie-blanche** [reply]
Hey, Emily! I really appreciate that you pointed out how there needs to be more of an emphasis on community and relationship building – which reminds me of Albert's emphasis on the importance of ethics in this class! As we learned last week, critical cartography has much of its systems rooted within a power dynamic that's based on privilege and power. By shifting the focus back to the community, marginalized groups that are disproportionately affected have a chance to be represented and finally have their voices heard. It is important for us to realize our positions of power and how we can use our standing to help those that are silenced.

**Comment by hrfrahn** [comment]
The Chambers article emphasizes the enormous emancipatory potential of Participatory Mapping; He documents a wide array of areas in which Participatory Mapping has been used effectively, from health to education to crime prevention. However, because the article was published in 2006, Chambers doesn't mention an application of Participatory Mapping that is perhaps the most commonly used among young people in America: apps like Google Maps and Waze that rely on crowdsourced data to provide information about travel time, traffic, restaurant reviews, and so much more. These apps are undeniably useful, mostly because they allow us to use the power of crowds to come to a consensus on questions that a single user might be unable to answer (is a restaurant good? how long will this drive take?). However, these apps are all run by private, for-profit businesses, raising questions about their real commitment to providing unbiased crowdsourced data if it interferes with their profits. I think civil society would be better served if apps like these operated on more of a "Wikipedia" model of crowdsourcing, where there's no pressure to make money; this would allow users to still benefit from the wisdom of their communities, but without ceding any control of the data to huge companies with questionable motivations.

**Comment by ronnie-blanche** [comment]
One way mapping and critical cartography be improved is through further deconstructing the power hierarchy within these systems. As pointed out in "An introduction to Critical Cartography", "the ability to make a map, even a stunning interactive 3D map, is now available to anyone with a home computer and an internet connection" (12). However, access to computers and the internet is a privilege that many people do not have – especially within marginalized communities, such as indigenous reservations. These communities not only are disproportionately negatively affected by the emphasis of power, but also suffer from erasure as map making and critical cartography have become entangled with academia and Western standards. Because of this, their maps are no longer considered maps just because they're "different," which leads to an even larger gap in power and privilege.

**Reply by ramirezirvin** [reply]
Totally agree, a good portion of these communities probably have been affected through the disadvantages they may face. Not everyone is able to have the same resources to view their updated work, some may not have the technological means to view any of this information. Some communities don't have access to view one's improved work leisurely through the internet and live a life without tech.

**Comment by ramirezirvin** [comment]
Participatory mapping and critical cartography can be improved by removing scholars and academics from improving it themselves and leaving it to the locals who originally built it. They could possibly help in providing resources to improve it, though it is best fit for people within the community to improve it through their means. As some communities do not have a lot of resources, some possibly have no access to technology as it could be due to how they may live their life. If it is being improved by outside sources (and possibly without their consent), can it still be claimed by the community locals as they essentially took their work from them and are disregarded in terms of improving their work without consent?

I also suggest spreading awareness and conserving their work, possibly through periodically visiting and seeing their improvements. Maybe updating it for others to be informed through a digital collective of some sort or spatial mapping for the academics and scholars. They can possibly work with the locals within the community to help them improve it through their resources means well where they can update and conserve their work at the same time.
